code/game/objects/structures/stool_bed_chair_nest/wheelchair.dm

```diff
@@ -28,57 +30,60 @@
 	var/calculated_move_delay
 	calculated_move_delay += 2 //wheelchairs are not infact sport bikes
 
-	if(buckled_mob)
-		if(buckled_mob.incapacitated())
-			return 0
-
-		var/mob/living/thedriver = user
-		var/mob_delay = thedriver.movement_delay()
-		if(mob_delay > 0)
-			calculated_move_delay += mob_delay
-
-		if(ishuman(buckled_mob))
-			var/mob/living/carbon/human/driver = user
-			var/obj/item/organ/external/l_hand = driver.get_organ("l_hand")
-			var/obj/item/organ/external/r_hand = driver.get_organ("r_hand")
-			if(!l_hand && !r_hand)
-				return 0 // No hands to drive your chair? Tough luck!
-
-			for(var/organ_name in list("l_hand","r_hand","l_arm","r_arm"))
-				var/obj/item/organ/external/E = driver.get_organ(organ_name)
-				if(!E)
-					calculated_move_delay += 4
-				else if(E.status & ORGAN_SPLINTED)
-					calculated_move_delay += 0.5
-				else if(E.status & ORGAN_BROKEN)
-					calculated_move_delay += 1.5
-
-		if(calculated_move_delay < 4)
-			calculated_move_delay = 4 //no racecarts
-
-		move_delay = world.time
-		move_delay += calculated_move_delay
-
-		if(!buckled_mob.Move(get_step(buckled_mob, direction), direction))
-			loc = buckled_mob.loc //we gotta go back
-			last_move = buckled_mob.last_move
-			inertia_dir = last_move
-			buckled_mob.inertia_dir = last_move
-			. = 0
-
-		else
-			. = 1
+	if(has_buckled_mobs())
+		for(var/m in buckled_mobs)
```

Comment:
This is just silly. You should probably just hardcode it to use whatever the first mob in the list is.